docker/images/pinot/Dockerfile

```diff
@@ -79,6 +79,9 @@ COPY bin ${PINOT_HOME}/bin
 COPY etc ${PINOT_HOME}/etc
 COPY examples ${PINOT_HOME}/examples
 
+RUN wget -O ${PINOT_HOME}/etc/jmx_prometheus_javaagent-0.16.1.jar https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.16.1/jmx_prometheus_javaagent-0.16.1.jar && \
```

Comment:
For backward compatibility, can we also download javaagent-0.12.0 as well?
So users on the latest image won't fail suddenly during restart.

Reply:
Great idea

Reply:
done